test/widgets/dialog_test.dart

```diff
@@ -7,7 +7,6 @@ import 'package:url_launcher/url_launcher.dart';
 import 'package:zulip/model/settings.dart';
 import 'package:zulip/widgets/app.dart';
 import 'package:zulip/widgets/dialog.dart';
-import 'package:zulip/widgets/store.dart';
```

Comment:
dialog test [nfc]: Remove an unused import
    
    Looks like this slipped through CI, introduced recently in
    8d3c1284e.

Ah, oops. I guess that changed in your revision of #1782, and I merged before CI finished running.

… In fact I'm pretty sure I ran `flutter test` on this very file before merging, because I saw CI wasn't done. But I guess I should have checked the analyzer output too.